sdk/packages/eternum/src/constants/structures.ts

```diff
@@ -180,7 +179,7 @@ export function determineEntityState(
 }
 
 export const STRUCTURE_COSTS: ResourceInputs = {
-  [StructureType.Hyperstructure]: HYPERSTRUCTURE_CREATION_COSTS,
+  [StructureType.Hyperstructure]: [], //todo hyperstructure costs
```

Comment:
todo?

Reply:
@aymericdelab does that need to be updated? i just removed it so it would compile